src/components/shared/ActivityFeed.tsx

```diff
@@ -1,36 +1,77 @@
 import * as React from "react";
```

Comment:
Can we leave quick comments that describe each component at the top?

Reply:
donezoes